docs/histogram-analysis/workflow.md

```diff
@@ -115,6 +115,7 @@ The relative population
 It can be estimated from the histogram by integrating each peak to a value 
 [*S*<sub>*j*</sub>](){: .math_var } and calculating the ratio:
 
+{: .equation }
 <img src="../assets/images/equations/HA-eq-pop.gif" alt="X_{j} = \frac{S_{j}}{\sum_{j'=1}^{J}( S_{j'})}">
```

Comment:
Ok. I need to understand, why equations are implemented as gif files.

Reply:
I did not find a better way than generating image files to display proper equations in html. For this I use the online tool at https://www.codecogs.com/latex/eqneditor.php